pom.xml

```diff
@@ -9,7 +9,7 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <jdk.version>11</jdk.version>
+        <jdk.version>1.8</jdk.version>
```

Comment:
why do we need to change this version?

Reply:
in previous homework it caused problems. command mvn clean package gave me errors. so in pom file i changed these two settings and it worked. in this home work i made it from the start